src/common/navigation.yaml

```diff
@@ -54,6 +54,7 @@ sections:
           - path: /network
           - path: /stacking
           - path: /local-development
+          - path: /technical-specs
```

Comment:
it might make sense to add a link to this new page from within the stacks 2.0 overview: https://github.com/blockstack/docs/blob/master/src/pages/stacks-blockchain/overview.md#features

Reply:
Done